scanner/Makefile

```diff
@@ -144,6 +138,11 @@ copy-db-scripts: ../image/postgres/download.sh
 	$(SILENT)echo "+ $@"
 	$(SILENT)cp $^ image/db
 
+.PHONY: copy-db-image-scripts
+copy-db-image-scripts: ../image/postgres/scripts/init-entrypoint.sh
+	$(SILENT)echo "+ $@"
+	$(SILENT)cp $^ image/db/scripts
+
```

Comment:
Nitpick: I haven't realized you wanted to keep the same script structure. We could put everything under `scripts/`, for example (like moving the ref to `scripts/download.sh` in the Dockerfile). Nonetheless, if you want to keep the original structure, this could work:

```suggestion
.PHONY: copy-db-scripts

# Files to copy.
copy-db-scripts := scripts/init-entrypoint.sh download.sh

# Directories.
copy-db-src     := ../image/postgres/
copy-db-dest    := image/db/

copy-db-scripts: $(addprefix $(copy-db-src),$(copy-db-scripts))
	$(SILENT)echo "+ $@"
	$(SILENT)tar -C $(copy-db-src) -cf - $(patsubst $(copy-db-src)%,./%,$?) \
	    | tar -C $(copy-db-dest) -xf -
```

Not tested since I am just nitpicking. It uses `tar` for compatibility, `cp` has `--parents`, but that would not run outside GNU/Linux.

Reply:
>  haven't realized you wanted to keep the same script structure
I'm indifferent on changing the structure, I just assumed keep the existing structure used by `central-db` and `scanner-db` today to not 'rock the boat' unnecessarily.